doc/Overview.md

```diff
@@ -19,37 +19,153 @@
+
+In C++, we need the headers of the model, the urdf parser and the forward-kinematics algorithms. In Python, Pinocchio and NumPy (numerics for Python) contexts are needed.
+
+The model is built by parsing the urdf model: the .urdf file should be directly provided to the parser. Here we do not parse (yet) the geometries, i.e. the shape of the bodies, but only the kinematic tree with names, lengths, inertias. The Data object is then built from the Model object, like in the previous example.
+
+The forward kinematics is simply computed by providing the model, the associated data and the robot configuration. Then, a loop goes through the kinematic tree (from root to leaves) and display for each joint its name and position in world frame. For the 6-dof manipulator UR5, there are 7 joints to be displayed, with the first joint being the "universe" i.e. the reference world frame from which everything is expressed.
+
+\section OverviewPython About Python wrapings
+
+Pinocchio is written in C++, with a full template-based C++ API, for efficiency purpose. All the functionalities are available in C++. Extension of the library should be preferably in C++.
+
+However, C++ efficiency comes with a higher work cost, especially for newcomers. For this reason, all the interface is exposed in Python. We tried to build the Python API as much as possible as a mirror of the C++ interface. The major difference is that the C++ interface is proposed using Eigen objects for matrices and vectors, that are exposed as Numpy Matrix in Python.
+
+When working with Pinocchio, we often suggest to first prototype your ideas in Python. Both the auto-typing and the scripting make it much faster to develop. Once you are happy with your prototype, then translate it in C++ while binding the API to have a mirror in Python that you can use to extend your idea. 
+
+\section OverviewCite How to cite Pinocchio
+
+Happy with Pinocchio? Please cite use with the following format.
+
+### Easy solution: cite our paper available in HAL
+The Pinocchio C++ library -- A fast and flexible implementation of rigid body dynamics algorithms and their analytical derivatives, by  J. Carpentier, G. Saurel, G. Buondonno, J. Mirabel, F. Lamiraux, O. Stasse and N. Mansard, in HAL ref 01866228
+
+\@article{carpentier-sii19, \n
+&nbsp;&nbsp;  title={The Pinocchio C++ library--A fast and flexible implementation of rigid body dynamics algorithms and their analytical derivatives},\n
+&nbsp;&nbsp; author={J. Carpentier and G. Saurel and G. Buondonno and J. Mirabel and F. Lamiraux and O. Stasse and N. Mansard},\n
+&nbsp;&nbsp; year={2018}\n
+}
+
+
+### Citing the software package
+
+Pinocchio: fast forward and inverse dynamics for poly-articulated systems, 2015–2018, by J. Carpentier, J. Mirabel, N. Mansard et al, at https://stack-of-tasks. github.io/pinocchio
```

Comment:
This is not the correct citation. The correct one is inside the Readme.md

Reply:
We should also update the README.md. 
I do not think the contribution of Florian put him in the top 3 dev of the current release of Pinocchio. Any suggestion of fairly deciding the contributor list is welcome. We should maybe remove direct naming of contributors in such a reference, as it would always be unfair, and have a page detailing the contribution in the doc.
I open an issue to discuss it.

Reply:
See related issue. As a side note, I put each citation in a separate tiny bib file, you can find them under doc/bib. If you want to modify this one, it is doc/bib/pinocchioweb.bib